Makefile

```diff
@@ -8,4 +8,7 @@ ROOT=$(realpath $(dir $(lastword $(MAKEFILE_LIST))))
 // TODO: add lint and format to github ci
 lint:
 	which golangci-lint || (go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.54.0)
-	golangci-lint run --config=$(ROOT)/.golangci.yml $(ROOT)/...
\ No newline at end of file
+	golangci-lint run --config=$(ROOT)/.golangci.yml $(ROOT)/...
+
+test:
+	go test -v ./...
```

Comment:
I guess the indentation with `4 spaces` is enough.

Reply:
I use one tab and its 4 space. in the `main`  branch that you write it its the same spacing

![image](https://github.com/ormushq/ormus/assets/96613482/eb13d745-db60-4cb4-828d-91a9472349cb)